### PR TITLE
Bug fixes in plotting functions and new demo notebook

### PR DESCRIPTION
**Pull request by radicamc**
Hi Joe,
As per my email, I've forked the repo and implemented the fixes to bugs that I have so far found - mostly in the plotting functionalities (for example I've fixed the issue #54 I filed, as well as an issue with unit selection in plot_spectrum). 
I also gave a rework to the example notebook you provided. I found that the original notebook was not running completely when I first started playing with awesimsoss's functionalities. I've updated it to fix a few bugs I found and make some walkthrough a little more explicit (in my opinion), I've ensured that it runs both on OS X and linux machines. I'm very much a Github noob, so I hope I've followed the proper procedures with this!
Cheers!

### CODE REVIEW

**Comment by radicamc** [comment]
I'm not sure of the procedure here, is there anything else I need to do before the repos can be merged? Apparently merging can only be completed by authorized users, of which I am not one!

**Comment by radicamc** [comment]
Added an additional patch to fix issue #58. Simulation outputs are now compatible with JWST stage 1 pipeline.

File: awesimsoss/awesim.py [hunk 1]
```diff
@@ -134,8 +134,8 @@ def __init__(self, ngrps, nints, star=None, planet=None, tmodel=None, snr=700,
         self.nints = nints
         self.nresets = nresets
         self.nframes = (self.nresets+self.ngrps)*self.nints
-        self.obs_date = datetime.datetime.now().strftime('%x')
-        self.obs_time = datetime.datetime.now().strftime('%X')
+        self.obs_date = datetime.datetime.now().strftime("%d/%m/%y")
```
Comment (hover2pi) [inline awesimsoss/awesim.py]: Actually, I think it needs to be `'%Y-%m-%d'`. Thanks for catching this!

File: awesimsoss/awesim.py [hunk 2]
```diff
@@ -680,7 +682,7 @@ def plot_lightcurve(self, column, time_unit='s', resolution_mult=20, draw=True):
         # Get the scaled flux in each column for the last group in
         # each integration
         flux_cols = np.nansum(self.tso_ideal.reshape(self.dims3)[self.ngrps-1::self.ngrps], axis=1)
-        flux_cols = flux_cols/np.nanmax(flux_cols, axis=1)[:, None]
+        flux_cols = flux_cols=flux_cols/np.nanmax(flux_cols, axis=0)[None, :]
```
Comment (hover2pi) [inline awesimsoss/awesim.py]: You've got an extra `flux_cols=` in there.

File: awesimsoss/awesim.py (hunk 1 above)
Comment (hover2pi) [inline awesimsoss/awesim.py]: Actually, I think it needs to be `%Y-%m-%d`. Thanks for catching this!

Comment (hover2pi) [reply awesimsoss/awesim.py]: Actually... is it `%d/%m/%y`? Is this in the documentation or is it just the only way it runs through the pipeline?

Comment (hover2pi) [reply awesimsoss/awesim.py]: Oh, just saw #58. Ok, nevermind!

Comment (hover2pi) [reply awesimsoss/awesim.py]: Though the description the FITS file stores in the header reads "[yyyy-mm-dd] UTC date at start of exposure" so should it be `%Y-%m-%d`?

Comment (radicamc) [reply awesimsoss/awesim.py]: I'm not sure, I'll run a test and see whether the pipeline complains if the format is ```%Y-%m-%d```. The pipeline definitely works with the ```%d/%m/%y``` format.

Comment (radicamc) [reply awesimsoss/awesim.py]: Just tested: simulations are compatible with the stage 1 pipeline if the OBS_DATE header keyword is in ```%Y-%m-%d``` format as well. Given that the header description lists this format, its probably the best option.

**Pull request merged**